BrainPortal/app/controllers/nh_loris_hooks_controller.rb

```diff
@@ -28,20 +28,29 @@ class NhLorisHooksController < NeurohubApplicationController
   Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
 
   before_action :login_required
-  api_available :file_list_maker
+  api_available :only => %i( file_list_maker csv_data_maker )
 
   # POST /loris_hooks/file_list_maker
+  #
+  # Receives:
+  #  {
+  #    source_basenames:  [ "abc.nii.gz", "def.mnc.gz" ],
+  #    source_data_provider_id:  123, # optiona, can be name
+  #    result_filenname:  "hello.csv",
```

Comment:
```suggestion
  #    result_filename:  "hello.csv",
```

Reply:
Yeah, I've seen these typos, will fix in a future commit